packages/terra-date-picker/src/react-datepicker/month.jsx

```diff
@@ -8,30 +8,105 @@ const FIXED_HEIGHT_STANDARD_WEEK_COUNT = 6
 
 export default class Month extends React.Component {
   static propTypes = {
+    /**
+     * Date Value Entered.
+     */
     day: PropTypes.object.isRequired,
+    /**
+     * A callback function for custom day class names.
+     */
     dayClassName: PropTypes.func,
+    /**
+     * Maximum Date value for a date range.
+     */
     endDate: PropTypes.object,
+    /**
+     * Array to store values of date that are disabled to pick .
+     */
     excludeDates: PropTypes.array,
+    /**
+     * Filter specific dates that can be selected only .
+     */
     filterDate: PropTypes.func,
+   /**
+     * Specifies whether the height of calendar dom fixed or variable.
+     */
     fixedHeight: PropTypes.bool,
+    /**
+     * A callback function to format week number .
+     */
     formatWeekNumber: PropTypes.func,
+    /**
+     * Highlight range of dates with custom classes.
+     */
     highlightDates: PropTypes.instanceOf(Map),
+    /**
+     * Show dates only in the given array.
+     */
     includeDates: PropTypes.array,
+    /**
+     * Prop to show inline version of date picker component.
+     */
     inline: PropTypes.bool,
+    /**
+     * Maximum Value of date that can be selected by user.
+     */
     maxDate: PropTypes.object,
+    /**
+     * Minimum Value of date that can be selected by user.
+     */
     minDate: PropTypes.object,
+    /**
+     * A callback function to execute when a date is selected.
+     */
     onDayClick: PropTypes.func,
+    /**
+     * A callback function to execute when day is clicked by mouse.
```

Comment:
This should be "A callback function to execute when the mouse cursor hovers over the day."